Training/Easy/chuck_norris.rb

```diff
@@ -27,25 +27,24 @@ def split_series(array)
     result
   end
 
-  def norris_parse(array)
-    result = ''
-    array.each do |a|
-      if a[0].zero?
-        result += '00 '
-      else
-        result += '0 '
-      end
-      result += '0' * a[1] + ' '
+  def norris_parse(series_array)
+    parsed = series_array.each_with_object('') do |a, result|
+      result << if a[0].zero?
+                  '00 '
```

Comment:
@vasil-gochev you mean something like this ? 
`      serie[0].zero? ? result << '00' : result << '0'
`